test/Operators/biops.js

```diff
@@ -46,10 +46,19 @@ var biops = [
     "&&", "||"                    // 11.11 Binary logical operators    
 ];
 
+function fixDate(output)
+{
+    if (output instanceof Date)
+    {
+        return output.toUTCString();
+    }
+    return output;
+}
+

```

Comment:
This test which used getVarDate was also marked as exclude_jenkins due to outputting dates to compare against a baseline - which was timezone dependent - I've tried to fix that here.